md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ArchiveConsumpQueueAPI.java

```diff
@@ -92,7 +92,7 @@ RestWrapper get(
             restWrapper = new RestWrapper(archiveConsumpQueue, RestWrapper.OK);
             LOGGER.info("Record with ID:" + queueId + " selected from ArchiveConsumpQueue by User:" + principal.getName());
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)